### DISCUSSION TITLE
SocialLogin

### DISCUSSION THREAD

**Discussion by GUmmanaveen** [abacritt/angularx-social-login#673] [Q&A]
I tried using the @abacritt/angularx-social-login library as @angularx-social-login is being deprecated. but its throwing an error stating that its expecting 6 to 8 parameters. Generic type 'ɵɵDirectiveDeclaration' requires between 6 and 8 type arguments.

15     static ɵdir: i0.ɵɵDirectiveDeclaration<GoogleSigninButtonDirective, "asl-google-signin-button", never, { "type": "type"; "size": "size"; "text": "text"; "shape": "shape"; "theme": "theme"; "logo_alignment": "logo_alignment"; "width": "width"; "locale": "locale"; }, {}, never, never, false, never>;.

Did anyone faced this one? How to resolve this issue.

**Comment by JuliaHorokhovatska** [comment]
same problem even though everything was fine before

**Reply by JuliaHorokhovatska** [reply]
help please!

**Reply by Splitzer26** [reply]
Same problem! I'm using Angular 14.0.0 in my project and the same problem.

**Reply by JuliaHorokhovatska** [reply]
I also!
